sim/priest/shadow/spriest.go

```diff
@@ -130,92 +131,86 @@ func (spriest *ShadowPriest) Reset(sim *core.Simulation) {
 	spriest.swStacks = 0
 }
 
+// TODO: probably do something different instead of making it global?
+const (
+	mbidx int = iota
+	swdidx
+	vtidx
+	swpidx
+)
+
 func (spriest *ShadowPriest) Act(sim *core.Simulation) time.Duration {
+	// fmt.Printf("[%0.1f] Priest Acting...\n", sim.CurrentTime.Seconds())
+	if spriest.rotation.PrecastVt && sim.CurrentTime == 0 {
```

Comment:
Why do we need a precast VT? Doesn't seem possible to cast before entering combat.

Reply:
basically simulating starting a cast while tank is running at the boss. Since the cast causes no threat (no immediate damage) and the next cast is also just a dot, many spriest will actually open the fight with VT landing at start of fight.

Reply:
(VT cast time is 1.5s long)